src/Filter/Factory/ToE164Factory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber\Filter\Factory;
+
+use Laminas\I18n\PhoneNumber\Factory\ConfigurationTrait;
+use Laminas\I18n\PhoneNumber\Filter\ToE164;
+use Psr\Container\ContainerInterface;
+
+final class ToE164Factory
+{
+    use ConfigurationTrait;
```

Comment:
Discussed above: let's avoid using the trait, and rather use static methods instead